### ISSUE TITLE
Shall surv.coxph have property "missings"?

### ISSUE THREAD

**Issue by MariaErdmann** [mlr-org/mlr#933]
Hi,

preparing a script that tests all survival learners I stumbled over the coxph learner, which has property "missings". Coxph as it is implemented now in mlr handles NAs in following manner: Calling predict() function reveals that for the according observation a NA is output (instead of a prediction). Therefore, the default performance measure, cindex, cannot be calculated. Hence, shall surv.coxph have property "missings"? Or am I missing something (implemented something wrong?)?
Below a code for reproduction:
````
## Preparation: either load_all() or the helper_objects need to be sourced after loading mlr library
load_all()
#library(mlr)
#source("helper_objects.R")

# Create data with missing value
data = getTaskData(surv.task)
data[1,3] = NA
task = makeSurvTask(data = data, target = surv.target)
head(getTaskData(task)) # x1 has a missing value in first obs

# surv.coxph has property "missings"
listLearners("surv", properties = "missings")$class

# Create learner, train, and predict
lrn = makeLearner("surv.coxph", par.vals = list(ties = "efron"))
m = train(lrn, task)
# But in the predict function nothing sophisticated happens with the missing value
p = predict(m, task)
perf = performance(p)
# cindex cannot be calculated
````

**Comment by berndbischl** [comment]
In this case remove the missing property from coxph now. And make a note in your general unit tests for learners-all to test theses cases. That learners that claim to work with missing do indeed work.

**Comment by berndbischl** [comment]
fixed in #934
